process/block/alteredaccounts/tokensProcessor.go

```diff
@@ -13,6 +13,13 @@ const (
 	idxTokenIDInTopics         = 0
 	idxTokenNonceInTopics      = 1
 	idxReceiverAddressInTopics = 3
+	issueFungibleESDTFunc      = "issue"
```

Comment:
duplicated declarations with the elrond-go-core/core package?

Reply:
I cannot find them declared elsewhere